integration-tests/input-customer-data.jmx

```diff
@@ -11,6 +11,16 @@
       <stringProp name="TestPlan.user_define_classpath"></stringProp>
     </TestPlan>
     <hashTree>
+      <Arguments guiclass="ArgumentsPanel" testclass="Arguments" testname="User Defined Variables" enabled="true">
```

Comment:
You can ignore all of this file.

I started to try to automate logging in, however, I didn't want to put a google password into my test, so I ended up just disabling what I started to do (prompt for google token) and instead put in a token I grabbed from a successful login attempt -- which will expire within a week.